suzieq/cli/sqcmds/command.py

```diff
@@ -87,6 +87,10 @@ def __init__(
             # This happens because nubia strips off the trailing quote
             # if not followed by a blank
             query_str += '"'
+        if query_str.count("'") % 2 != 0:
```

Comment:
```suggestion
        elif query_str.count("'") % 2 != 0:
```
I think there is no need to to evaluate this expression if the previous one is true